app/assets/src/styles/samples.scss

```diff
@@ -1107,7 +1107,7 @@ input[type="button"] {
           padding-top: 20px;
         }
         .compare {
-          padding: 5px 30px;
+          padding: 5px;
```

Comment:
Why is the padding change required?

Reply:
It is temporary. Padding caused the text in the button to spread over 2 lines.

Reply:
removed padding change as part of switching to generic button